using-mobile-extensions/adobe-campaignclassic/adobe-campaignclass-c-api-reference.md

```diff
@@ -0,0 +1,261 @@
+# Campaign Classic API Reference
+
+## registerDevice API
+
+### Registering a user's device with Campaign Classic
+
+{% tabs %}
+
+{% tab title="Android" %}
+
+Follow [Google's instructions](https://firebase.google.com/docs/cloud-messaging/android/client) to get your app ready to handle push notifications. Once you receive the FCM SDK registration token, you will need to send that token along with the device information to Campaign Classic using the registerDevice API.
+
+### registerDevice
+
+The registerDevice API will register a device with your Campaign Classic registration server. It takes the FCM registration token as a parameter along with a user key that identifies the user such as an email address or login name. It can also be given a Map of custom key value pairs that you want to associate with the registration. Lastly, a boolean value is returned in the callback which signals whether the registration was sucessful.
```

Comment:
- In the sentence, please change "along with" to just "with".
- In the final sentence, please change from "Lastly" to "As a result"